Movielens 1m discussion (#88)
thesis20/report: sections/discussion/discussion-chapter.tex

We suspect this because the datasets that were used for the development of these methods are generally a lot larger.
For example, the smallest dataset used in the \textit{LightGCN}\cite{LightGCN} paper has about a million interactions.
Therefore it could be interesting to run the experiment on a larger dataset to see if the graph-based methods achieve better performance.
\\\\

> To test the if the size of the dataset matters for the graph-based methods, the experiment was run on the MovieLens dataset with one million interactions.

Reviewer: ```suggestion
To test whether the size of the dataset matters for the graph-based methods, the experiment was re-run on the MovieLens dataset with one million interactions.
```

> The results can be seen in %Ref til table i appendix?. 

Reviewer: ```suggestion
The results can be seen in \autoref{app:tables}.
```

The results were somewhat the same as was seen for MovieLens 100k.
None of the graph-based methods performed well on this dataset, and they only had comparable results with their NDCG@10 values.

> Therefore it can not be concluded that the size of the dataset is what makes the graph-based methods run poorly. 

Reviewer 2: ```suggestion
Therefore it cannot be concluded that the size of the dataset is what makes the graph-based methods run poorly. 
```

> Instead we suspect, as mentioned earlier, that the main reason they perform poorly is because they do not utilize the ratings like the other methods do, but are rather used for datasets with implicit ratings.

Reviewer: ```suggestion
Instead, we suspect, as mentioned earlier, that the main reason they perform poorly is that they do not utilize explicit ratings like the other methods do, but are rather used for datasets with implicit ratings.
```

> Instead, we suspect, as mentioned earlier, that the main reason they perform poorly is that they do not utilize explicit ratings like the other methods do, but are rather used for datasets with implicit ratings.

Reviewer 2: ref til der vi mentioner det?

Reviewer: discussion-chapter.tex linje 143 bntw

Author: ja det er næsten lige ovenover, behøver vi ref?

Reviewer 2: okay så undskyld altså

Author: jo tak plz stfu næste gang

Reviewer 2: :(

> Instead, we suspect, as mentioned earlier, that the main reason they perform poorly is that they do not utilize explicit ratings like the other methods do, but are rather used for datasets with implicit ratings.

Reviewer 2: ```suggestion
Instead, we suspect, as mentioned earlier, that the main reason they perform poorly is that they do not utilize explicit ratings like the other methods do, but are rather intended to be used for datasets with implicit ratings.
```
